api/pkg/backend/service.go

```diff
@@ -461,9 +461,117 @@ func (s *APIService) listStorageType(ctx context.Context, request *restful.Reque
 	return storageTypes, err
 }
 
+func (s *APIService) IsDuplicateItemExist(itemList []string) []string {
+	var dupList []string
+	visited := make(map[string]int, 0)
+	for _, val := range itemList {
+		if visited[val] >= 1 {
+			dupList = append(dupList, val)
+		}
+		visited[val]++
+	}
+	return dupList
+}
+
+func (s *APIService) CheckInvalidBackends(ctx context.Context, backends []string) ([]string, error) {
```

Comment:
https://github.com/sodafoundation/multi-cloud/pull/1276/files has GetInvalidBackends(). So either one should be kept

Reply:
Yeah, Since...here i have modified and tested with these values. We can kep this.